src/test/http-mocks/claim-store.ts

```diff
@@ -58,7 +58,7 @@ export const sampleClaimObj = {
       }
     },
     interest: {
-      type: InterestType.NO_INTEREST
+      type: InterestType.STANDARD
```

Comment:
if type is Standard don't we need more data in this object?

Reply:
It is there already. It seems like `InterestType.NO_INTEREST` wasn't used properly.